framework/src/setroubleshoot/server.py

```diff
@@ -115,6 +117,12 @@ def sighandler(signum, frame):
         return
 
 
+def polling_failled_handler(signum, frame):
+    log_debug("received signal=%s" % signum)
+    global continue_polling
+    continue_polling = False
+
```

Comment:
I'd rather see the same concept as is in the stable branch.
Given the problem in `selinux.audit2why`, `sys.exit()` doesn't seem to work in this case, but you can temporary force exit using `os._exit()`. Please also a FIXME comment with some explanation and reference to the audit2why problem similar to the bellow one.